packages/compass-home/src/components/workspace.tsx

```diff
@@ -41,6 +42,16 @@ export default function Workspace({
     typeof WorkspacesPlugin
   >['onActiveWorkspaceTabChange'];
 }): React.ReactElement {
+  // CompassSchemaPlugin requires leaflet which does not render outside of Electron
```

Comment:
Right, the point is that it doesn’t work outside of it, and yes, it fails in plain mocha … the error is this:

```
ReferenceError: L is not defined
    at /home/addaleax/src/compass/node_modules/leaflet-draw/dist/leaflet.draw.js:9:96
    at Object.<anonymous> (/home/addaleax/src/compass/node_modules/leaflet-draw/dist/leaflet.draw.js:11:3412)
    […]
    at require (node:internal/modules/helpers:130:18)
    at webpackUniversalModuleDefinition (/home/addaleax/src/compass/node_modules/react-leaflet-draw/dist/react-leaflet-draw.js:3:74)
    at Object.<anonymous> (/home/addaleax/src/compass/node_modules/react-leaflet-draw/dist/react-leaflet-draw.js:10:3)
    […]
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (/home/addaleax/src/compass/packages/compass-schema/src/components/coordinates-minichart/coordinates-minichart.jsx:7:1)
```

because `leaflet.draw.js` requires `L` from `leaflet` to be defined on `globalThis`, but `leaflet` itself installs itself on `window` (which are distinct in the jsdom-in-Node.js case).

Happy to take any other solution here! We could probably also just drop the Node.js `test` task and only use `test-electron` for compass-home, for example.

Reply:
It's kind of a pain to find now that we moved all jsdom mocha config to a separate repo, but this default jsdom mode of working is already not a thing that works for us, this is why we use [`global-jsdom` package](https://github.com/mongodb-js/devtools-shared/blob/e55cd86d06a1258bfb6da599fb00496b5acc3146/configs/mocha-config-devtools/register/jsdom-global-register.js#L3) (not just jsdom) and in some cases where `global-jsdom` is not up to date on stuff that needs to be re-assigned, we manually re-assigned some properties from `window` to `globalThis` ([like here for example](https://github.com/mongodb-js/devtools-shared/blob/e55cd86d06a1258bfb6da599fb00496b5acc3146/configs/mocha-config-devtools/register/jsdom-global-register.js#L56-L58)). I think it would be in line with this to add `L` to `globalThis` from `window` to follow that pattern. I guess as this is probably set on `window` after mocha register hooks run, we can do this by defining it as a getter?

Reply:
Yup, that works – dd88c7226b :)